src/github/handlers/index.ts

```diff
@@ -98,16 +98,21 @@ async function handleEvent(event: EmitterWebhookEvent, eventHandler: InstanceTyp
     state.inputs[0] = inputs;
     await eventHandler.pluginChainState.put(stateId, state);
 
-    if (!isGithubPluginObject) {
-      await dispatchWorker(plugin, await inputs.getWorkerInputs());
-    } else {
-      await dispatchWorkflow(context, {
-        owner: plugin.owner,
-        repository: plugin.repo,
-        workflowId: plugin.workflowId,
-        ref: plugin.ref,
-        inputs: await inputs.getWorkflowInputs(),
-      });
+    // We wrap the dispatch so a failing plugin doesn't break the whole execution
+    try {
+      if (!isGithubPluginObject) {
+        await dispatchWorker(plugin, await inputs.getWorkerInputs());
```

Comment:
Just curious but does it make sense to Promise.all and invoke all requests concurrently?

Alternatively we should make the ordering more explicit in our config so that its more apparent we intend to take outputs from one and make it accessible to the others? 

The concept of plugin chains is useful but seems very abstract with our current config layout. Hence why im inquiring about concurrency. 

It could be interesting to support in series for configs that explicitly need that, and concurrency by default?

Reply:
We could in here yes, I can change it. The order here doesn't matter, since the chaining is not handled in this part of the code but in the callbacks.